src/main/java/jUnitTask/QEquation.java

```diff
@@ -0,0 +1,67 @@
+package jUnitTask;
+
+/**
+ * Hаписать тесты для класса решения квадратного уравнения
+ *
+ * Добавить тесты с параметрами
+ */
+public class QEquation {
+    public static Pair qEquation(double a, double b, double c) {
+        double x1;
+        double x2;
+        if (a == 0) {
+            throw new ArithmeticException("Division by zero");
+        }
+        x1 = (-b + Math.sqrt(Math.pow(b, 2) - 4 * a * c)) / (2 * a);
+        x2 = (-b - Math.sqrt(Math.pow(b, 2) - 4 * a * c)) / (2 * a);
+        return new Pair(x1, x2);
+    }
+
+    public static void main(String[] args) {
```

Comment:
It's better to move exec method to separate class. To have separate util class and separate exec class